### DISCUSSION TITLE
Limiting memory utilization while using `to_parquet()`

### DISCUSSION THREAD

**Discussion by ryandstoughton** [ibis-project/ibis#7957] [Q&A]
Using `to_parquet()` to generate a parquet file from a db query seems to pull the entire result into memory. Is there a way to stream this data? Using `to_parquet()` becomes prohibitively expensive for larger queries.

**Comment by lostmygithubaccount** [comment]
hi @ryandstoughton, which backend(s) are you using and can you provide an example of the code you are running?

**Comment by cpcloud** [comment]
Closing as stale.
